### PR TITLE
Map DotNetRestoreSources => RestoreSources for metapackage builds

### PR DESCRIPTION
**Pull request by natemcmaster**

### CODE REVIEW

File: build/sources.props [hunk 1]
```diff
@@ -1,11 +1,18 @@
 <Project>
+
+  <Import Project="$(DotNetRestoreSourcePropsPath)" Condition="'$(DotNetRestoreSourcePropsPath)' != ''"/>
+
   <PropertyGroup>
     <RestoreSources>$(DotNetAdditionalRestoreSources)</RestoreSources>
     <RestoreSources>
+      $(RestoreSources);
```
Comment (JunTaoLuo) [inline build/sources.props]: Nit combine with the line above

**Comment by natemcmaster** [comment]
I did quick validation. The error message before says:
```
:45]C:\b\w\33bdfc1cae7b2a38\.w\Microsoft.AspNetCore.App\Microsoft.AspNetCore.App.csproj : error NU1102: Unable to find package Microsoft.AspNetCore.NodeServices with version (= 2.1.0-preview2-30312)
[11:41:45]C:\b\w\33bdfc1cae7b2a38\.w\Microsoft.AspNetCore.App\Microsoft.AspNetCore.App.csproj : error NU1102:   - Found 33 version(s) in nuget.org [ Nearest version: 2.1.0-preview1-final ]
```
With this change it shows that the additional restore sources are available.
image

**Comment by ryanbrandenburg** [comment]
Sounds good to me.

File: build/sources.props (hunk 1 above)
Comment (natemcmaster) [reply build/sources.props]: updated

**Pull request merged**